docs/docker_container.md

```diff
@@ -82,12 +82,11 @@ as well as a release package (.tar.gz, with ovms binary and necessary libraries)
 Use command `make docker_build BASE_OS=redhat` or `make docker_build BASE_OS=centos`. OVMS with ubi base image doesn't support NCS and HDDL accelerators.
 
 Additionally you can set version of GPU driver used by the produced image. Currently following versions are available:
-- 19.41.14441
-- 20.35.17767
+- 21.49.21786
```

Comment:
Can we add also the version for Ubuntu and specify which one works with which system?